src/app/CodeRepo/lib/backends/github/index.js

```diff
@@ -118,28 +119,37 @@ class GithubBackend extends AsyncEventEmitter {
         };
     }
 
-    // TODO: Find a better way for this, resolving is dependant on user having pushed in last 30 events
-    // This is a shot at resolving the username on a review by listing the user events
-    async _getUserEmail(userName) {
-        const url = `${GITHUB_API_BASE}/users/${userName}/events/public`;
-        try {
-            const result = await this._sendRequest(url, {}, "GET");
-            for (const event of result) {
-                if (event.type && event.type === "PushEvent" && event.payload && event.payload.commits) {
-                    for (const commit of event.payload.commits) {
-                        if (commit.author.name === userName) {
-                            return commit.author.email;
-                        }
-                    }
+    // Lookup user based on github alias
+    async _getUserByGithubLogin(githubUserName) {
+        let ref=false
+
+        if (typeof githubUserName === "string" && githubUserName.length > 0) {
```

Comment:
Maybe it's just style but if we do error cases first and return, we don't get so deeply nested { which in my view makes things harder to read.